runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomConfig.java

```diff
@@ -72,14 +72,37 @@ default boolean relaxCameraPitch()
 		return false;
 	}
 
+
 	@ConfigItem(
-		keyName = "requireControlDown",
-		name = "Require control down",
-		description = "Configures if holding control is required for zooming",
-		position = 4
+		keyName = "controlFunction",
+		name = "Control Function",
+		description = "Configures the zoom function when control is pressed",
+		position = 5
 	)
-	default boolean requireControlDown()
+	default ControlFunction controlFunction()
 	{
-		return false;
+		return ControlFunction.NONE;
+	}
+
+	@ConfigItem(
+		keyName = "ctrlZoomValue",
+		name = "Reset zoom position",
+		description = "Position of zoom when it is reset",
+		position = 6
+	)
+	default int ctrlZoomValue()
+	{
+		return 600;
```

Comment:
I'm not sure this needs to be configurable or not, but it should at least default to the default zoom level which I believe is 512?